src/to_mathlib/analysis/inner_product_space/tensor_product.lean

```diff
@@ -28,118 +87,12 @@ linear_map.mul' ℝ ℝ ∘ₗ
   tensor_product_aux E F ((e₁ ⊗ₜ[ℝ] f₁) ⊗ₜ[ℝ] (e₂ ⊗ₜ[ℝ] f₂)) = ⟪e₁, e₂⟫ * ⟪f₁, f₂⟫ :=
 by simp [tensor_product_aux, as_tensor]
 
-/- We may not actually need this for this project since what we really want is the `instance`
-lower down for the _tensor power_.
-
-This is still a gap in Mathlib that is worth filling. -/
-
--- new stuff starts here (line 30 in my analysis/inner_product_space/tensor_product file)
-
-
-section some_linear_algebra_from_kevin
-
-local attribute [ext] tensor_product.ext
-
-/-- Two linear maps (M ⊗ N) ⊗ (P ⊗ Q) → S which agree on all elements of the
-form (m ⊗ₜ n) ⊗ₜ (p ⊗ₜ q) are equal. -/
-theorem ext_fourfold' {R M N P Q S: Type*} [comm_semiring R]
-  [add_comm_monoid M] [add_comm_monoid N] [add_comm_monoid P] [add_comm_monoid Q]
-  [add_comm_monoid S] [module R M] [module R N] [module R P] [module R Q] [module R S]
-  {φ ψ : (M ⊗[R] N) ⊗[R] (P ⊗[R] Q) →ₗ[R] S}
-  (H : ∀ w x y z, φ ((w ⊗ₜ x) ⊗ₜ (y ⊗ₜ z)) = ψ ((w ⊗ₜ x) ⊗ₜ (y ⊗ₜ z))) : φ = ψ :=
-begin
-  ext m n p q,
-  exact H m n p q,
-end
-
-end some_linear_algebra_from_kevin
-
-/-- We may not actually need this for this project since what we really want is the `instance`
-lower down for the _tensor power_.
+-- instance (k : ℕ) : inner_product_space ℝ (⨂[ℝ]^k E) := sorry
```

Comment:
We need to keep this instance for use elsewhere in the project
```suggestion
instance (k : ℕ) : inner_product_space ℝ (⨂[ℝ]^k E) := sorry
```

Reply:
re-added

Reply:
Given that you have no added this instance elsewhere we should remove this code, not comment it out.